openshift/templates/images/wordpress/docker/dev.Dockerfile

```diff
@@ -0,0 +1,14 @@
+# OpenShift uses version from buildconfig - Don't update
+FROM wordpress:6-php8.3-fpm-alpine
+# Install LDAP for Next Active Directory Integration Plugin.
+RUN set -ex; \
+        apk add --no-cache --virtual .build-deps \
+        openldap-dev; \
+        docker-php-ext-install ldap; \
+        docker-php-ext-enable ldap;
+
+COPY php.conf.ini /usr/local/etc/php/conf.d/php.ini
+
+# Required for installing the WP unit testing environment
+RUN apk add subversion
```

Comment:
technically you could run these as part of the main run on line 4

Reply:
True but at least this way the code that's different from the production Dockerfile is appended to the end instead of mixed in with the production code so it's easier to tell the difference between them.